app/workers/callbacks/http_callback.rb

```diff
@@ -0,0 +1,43 @@
+# encoding: utf-8
+
+require 'excon'
+
+module HttpCallback
+  def http_callback(web_hook)
+    http_execute(web_hook[:url], web_hook[:message])
+  end
+
+  alias_method :https_callback, :http_callback
+
+  def http_execute(uri, data, options={})
+    connection = Excon.new(uri.to_s, ssl_verify_peer: false )
```

Comment:
`ssl_verify_peer: false` is *extremely* dangerous in this context. I acknowledge that there's a good chance this is how this has always been, and getting ruby's OpenSSL to recognize the right CA bundle can be a pain, but I think this option should be removed because it offers a false sense of security.

If the certificate can't be verified against a trusted CA, it may as well be in plaintext, since an attacker in a position to read the plaintext version can (for just slightly more CPU) transparently proxy the SSL version and get at the plaintext that way.